root/static/scripts/release-editor/init.js

```diff
@@ -268,7 +268,7 @@ releaseEditor.init = function (options) {
     this.seed(options.seed);
 
     if (this.action === "edit") {
-        this.loadRelease(options.gid);
```

Comment:
`loadRelease` is still used when `action === "add"` to switch between release duplicates.